assets/js/base/components/totals/totals-coupon-code-input/index.js

```diff
@@ -59,29 +59,31 @@ const TotalsCouponCodeInput = ( {
 				isLoading={ isLoading }
 				showSpinner={ false }
 			>
-				<PanelRow className="wc-block-coupon-code__row">
-					<TextInput
-						id={ `wc-block-coupon-code__input-${ componentId }` }
-						className="wc-block-coupon-code__input"
-						label={ __(
-							'Enter code',
-							'woo-gutenberg-products-block'
-						) }
-						value={ couponValue }
-						onChange={ ( newCouponValue ) =>
-							setCouponValue( newCouponValue )
-						}
-					/>
-					<Button
-						className="wc-block-coupon-code__button"
-						disabled={ isLoading }
-						onClick={ () => {
-							onSubmit( couponValue );
-						} }
-						type="submit"
-					>
-						{ __( 'Apply', 'woo-gutenberg-products-block' ) }
-					</Button>
+				<PanelRow>
```

Comment:
Why was the class removed? `wc-block-coupon-code__row`

Reply:
I removed it because it became `.wc-block-coupon-code__form` in the inner form so the `PanelRow` didn't have any styles applied to it anymore. But I don't have anything against keeping the CSS class, indeed it might make theme styling easier, so adding it back in 8fd80c9ee718b92179ae4cde2240cfabcadf07b9. :+1: